skyve-war/src/main/java/modules/admin/Communication/Communication.xml

```diff
@@ -3,6 +3,7 @@
 			xmlns="http://www.skyve.org/xml/document"
 			xsi:schemaLocation="http://www.skyve.org/xml/document ../../../schemas/document.xsd"
 			xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<documentation>Manages bulk email communications with support for templates, attachments, calendar invites, and targeted recipient selection through tags or document bindings.</documentation>
```

Comment:
This should probably mention that it supports mail-merge type functionality, allowing Skyve bindings to be substituted into the subject and body. They can also be used for individual communications (the primary purpose), not bulk.